examples/vl3_basic/helm/vl3/values.yaml

```diff
@@ -3,22 +3,23 @@
 # Declare variables to be passed into your templates.
 
 registry: docker.io
-org: tiswanso
+org: mistrate
```

Comment:
don't change here... you will pass params to the helm command or scripts that call helm to override these values